### ISSUE TITLE
missing backport.h

### ISSUE THREAD

**Issue by wdehoog** [ubports/ubuntu-touch-for-oneplus-one#21]
When trying to build from source I get:

In file included from /home/neil/phablet/kernel/oneplus/msm8974/security/apparmor/apparmorfs.c:25:0:
/home/neil/phablet/kernel/oneplus/msm8974/security/apparmor/include/apparmor.h:21:22: fatal error: backport.h: No such file or directory

**Comment by DomThePorcupine** [comment]
Did you fix the manifest files? How did you get the repo to sync?

**Comment by wdehoog** [comment]
I assumed the remote 'opo' from roomservice.xml is the one to use so disabled the one in manifest.xml. 
(I also added --force-resync in autosetup.sh but I do not think that made a difference).

**Comment by DomThePorcupine** [comment]
Okay I'm getting the same error about the backport.h as well

**Comment by wdehoog** [comment]
I think the apparmor patch did not include all files or it included the wrong files. 

The remark at the bottom of page https://github.com/ubports/android_kernel_oneplus_one/commit/6778b16c2c50d3a960630daecd1275c4e662c036 links to an apparmor directory for another device. 

When I replaced my 'bacon' one whith it compilation still failed but on another error (duplicate definition if i remember well). By commenting out that function (iterate_fd in backport.c) the build seems to finish ok. 
I did not test the produced images in my phone (not am I planning to do so).
